object-manipulation-3/main.js

```diff
@@ -0,0 +1,98 @@
+// assign each player a pair of cards
+// log the players to the console to see their  cards
+// return the cards from the function.
+// deal the cards to each player
+// create a function to play the game and decide the winner
+// create a variable for the final number
+// create a variable with an empty string for the winner
+// create a variable to call the dealCards function and store the return
+// loop through the returned players array
+// assign the values of the pair to a variable to count the total score
+// check if its an ace, king, queen or jack and assign the value accordingly
+// add the two cards together and get the total points
+// check if the total points is greater than the previous sumTotal
+// if it is, replace the previous highest score with the newest
+// concactenate a string to declair the new winner
+// return the final winner and log it to the console.
+
+var cardPlayers = [
+  {
+    name: 'Player One',
+    hand: []
+  },
+  {
+    name: 'Player Two',
+    hand: []
+  },
+  {
+    name: 'Player Three',
+    hand: []
+  },
+  {
+    name: 'Player Four',
+    hand: []
+  }
+];
+
+var cards = [
+  { suite: 'clubs', rank: 'Ace' }, { suite: 'clubs', rank: 1 }, { suite: 'clubs', rank: 2 }, { suite: 'clubs', rank: 3 }, { suite: 'clubs', rank: 4 },
+  { suite: 'clubs', rank: 5 }, { suite: 'clubs', rank: 6 }, { suite: 'clubs', rank: 7 }, { suite: 'clubs', rank: 8 }, { suite: 'clubs', rank: 9 },
+  { suite: 'clubs', rank: 10 }, { suite: 'clubs', rank: 'Jack' }, { suite: 'clubs', rank: 'Queen' }, { suite: 'clubs', rank: 'King' },
+  { suite: 'Hearts', rank: 'Ace' }, { suite: 'Hearts', rank: 1 }, { suite: 'Hearts', rank: 2 }, { suite: 'Hearts', rank: 3 }, { suite: 'Hearts', rank: 4 },
+  { suite: 'Hearts', rank: 5 }, { suite: 'Hearts', rank: 6 }, { suite: 'Hearts', rank: 7 }, { suite: 'Hearts', rank: 8 }, { suite: 'Hearts', rank: 9 },
+  { suite: 'Hearts', rank: 10 }, { suite: 'Hearts', rank: 'Jack' }, { suite: 'Hearts', rank: 'Queen' }, { suite: 'Hearts', rank: 'King' },
+  { suite: 'Diamonds', rank: 'Ace' }, { suite: 'Diamonds', rank: 1 }, { suite: 'Diamonds', rank: 2 }, { suite: 'Diamonds', rank: 3 }, { suite: 'Diamonds', rank: 4 },
+  { suite: 'Diamonds', rank: 5 }, { suite: 'Diamonds', rank: 6 }, { suite: 'Diamonds', rank: 7 }, { suite: 'Diamonds', rank: 8 }, { suite: 'Diamonds', rank: 9 },
+  { suite: 'Diamonds', rank: 10 }, { suite: 'Diamonds', rank: 'Jack' }, { suite: 'Diamonds', rank: 'Queen' }, { suite: 'Diamonds', rank: 'King' },
+  { suite: 'Spades', rank: 'Ace' }, { suite: 'Spades', rank: 1 }, { suite: 'Spades', rank: 2 }, { suite: 'Spades', rank: 3 }, { suite: 'Spades', rank: 4 },
+  { suite: 'Spades', rank: 5 }, { suite: 'Spades', rank: 6 }, { suite: 'Spades', rank: 7 }, { suite: 'Spades', rank: 8 }, { suite: 'Spades', rank: 9 },
+  { suite: 'Spades', rank: 10 }, { suite: 'Spades', rank: 'Jack' }, { suite: 'Spades', rank: 'Queen' }, { suite: 'Spades', rank: 'King' }
+];
```

Comment:
Instead of writing these all out by hand, you should make an array containing all possible suits, and an array containing all possible ranks, and loop over those arrays to generate a deck.